lib/mapping/CMapHeader.h

```diff
@@ -223,8 +224,8 @@ class DLL_LINKAGE CMapHeader
 	si32 height; /// The default value is 72.
 	si32 width; /// The default value is 72.
 	bool twoLevel; /// The default value is true.
-	std::string name;
-	std::string description;
+	MetaString name;
+	MetaString description;
```

Comment:
I decided do not store raw ids to encapsulate the logic. Also, description may contain additional fields, such as author, license and other meta information

Reply:
True, but unless I am missing something, this also changes our Json format.
From simple "name" : "Map Name"
To rather complex format used by MetaString serializer

Reply:
you still can use "name" : "Map Name" and it will be just a raw string in this case. Backward compatibility ensured. Some overhead is present of course, but not sure if better to sacrifice type unification or extra serialisation complexity